src/browser/styles/app.scss

```diff
@@ -4,117 +4,30 @@ body {
     font-family: 'Roboto', sans-serif;
 }
 
-.nav-item {
-    padding: .5em 1em;
```

Comment:
Pulled out a lot of this css to a page-level file so that we don't have one large bloat of a file.